sdk/servicebus/service-bus/src/core/managementClient.ts

```diff
@@ -925,6 +925,70 @@ export class ManagementClient extends LinkEntity<RequestResponseLink> {
     }
   }
 
+  /**
+   * Batch-delete messages.
+   *
+   * @param messageCount - number of messages to delete in a batch.
+   * @param enqueueTimeUtcOlderThan - Delete messages whose enqueue time (UTC) are older than this.
+   * @returns number of messages deleted.
+   */
+  async batchDeleteMessages(
+    messageCount: number,
+    enqueueTimeUtcOlderThan?: Date,
+    sessionId?: string,
+    options: OperationOptionsBase & SendManagementRequestOptions = {},
+  ): Promise<number> {
+    throwTypeErrorIfParameterMissing(this._context.connectionId, "messageCount", messageCount);
+    throwTypeErrorIfParameterTypeMismatch(
+      this._context.connectionId,
+      "messageCount",
+      messageCount,
+      "number",
+    );
+
+    if (isNaN(messageCount) || messageCount < 1) {
+      throw new TypeError("'messageCount' must be a number greater than 0.");
```

Comment:
Do we need to do this? Is the error coming from the service not clear?

Reply:
I am following the existing pattern. Haven't checked the service error yet.